src/cli/cli-definitions.js

```diff
@@ -174,5 +174,15 @@ export default {
   "verbose": {
     env: "VERBOSE",
     help: "Set the logging to verbose"
+  },
+  "logLevel": {
+    help: "Sets the log level"
+  },
+  "enableConfigChanges": {
+    env: "ENABLE_CONFIG_CHANGES",
```

Comment:
Or even just removing this setting, and having all settings be editable unless defined in code/cli. The endpoint requires the master key anyway, so it wouldn't be a security issue, and you could always just define everything in code if you want to disable editing even for master key.